src/components/offer-card/OfferCard.jsx

```diff
@@ -0,0 +1,95 @@
+import { t } from 'i18next'
+
+import Rating from '@mui/material/Rating'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+import IconButton from '@mui/material/IconButton'
+import TurnedIn from '@mui/icons-material/TurnedIn'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+
+import AppButton from '../app-button/AppButton'
+import AppChip from '../app-chip/AppChip'
```

Comment:
Use import like this 
import AppButton from '~/components/app-button/AppButton'
import AppChip from '~/components/app-chip/AppChip'

Reply:
Done.